app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  def update_post_comment_counter
+    post.increment!(:comments_counter)
+  end
+
+  def update_post_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] REQUIRED CHANGE: Please delete this. It is a duplication of what is already being handled by the `like` model.